sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -59,8 +59,9 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
       sparkSession: SparkSession,
       options: Map[String, String],
       files: Seq[FileStatus]): Option[StructType] = {
+    val fileNames = files.map(_.getPath.toString)
```

Comment:
not a big deal but just curious - why did you name it `fileNames` BTW? I thought it's going to be just like `paths`.

Reply:
because they are `String` and not `Path`s. Actually this change is not needed, I changed it only because it was easier for me while debugging. I can put it back as it was before.

Reply:
I mean it's not the name of a file though.

Reply:
yes, I reverted the change, thanks.